.github/workflows/lint.yaml

```diff
@@ -41,7 +41,7 @@ jobs:
         run: |
           python3 -m pip install pylint
       - name: pylint
-        run: pylint * --ignore=old_examples --max-line-length=120
+        run: pylint */ *.py --ignore=old_examples --max-line-length=120
```

Comment:
`pylint **/*.py` does not work?

Reply:
it does, but then `old_examples` is not ignored, and I didn't think this version ugly enough to put more thought in

Reply:
Ah, did not think of that, good point!